misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/RawMaterialIssueStepButton.jsx

```diff
@@ -1,33 +1,27 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import counterpart from 'counterpart';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { withRouter } from 'react-router';
 
-import { pushHeaderEntry } from '../../../actions/HeaderActions';
-import StepButton from '../common/StepButton';
+import { manufacturingStepScreenLocation } from '../../../routes/manufacturing';
 import Indicator from '../../../components/Indicator';
 import * as CompleteStatus from '../../../constants/CompleteStatus';
 
 class RawMaterialIssueStepButton extends PureComponent {
   handleClick = () => {
-    const { locatorName, location, dispatch, onHandleClick } = this.props;
+    const { push } = this.props;
+    const location = manufacturingStepScreenLocation(this.props);
```

Comment:
pls send only the specific params.

PS: an IDE like IntelliJ or something like github copilot might help you to quickly write those things.